CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/CollectionRequestHandler.swift

```diff
@@ -0,0 +1,142 @@
+//RequestHandler.swift
+//CBLTestServer-iOS
+
+import Foundation
+import CouchbaseLiteSwift
+
+public class CollectionRequestHandler {
+    public static let VOID: String? = nil
+    fileprivate var _pushPullReplListener:NSObjectProtocol?
+    
+    public func handleRequest(method: String, args: Args) throws -> Any? {
+        switch method {
+        case "collection_defaultCollection":
+            let database: Database = (args.get(name:"database"))!
+            let collection: Collection = try (database.defaultCollection())!
+            return collection
+            
+        case "collection_getCollectionName":
+            let collection: Collection = (args.get(name:"collection"))!
+            return collection.name
+            
+        case "collection_createCollection":
+            let scopeName: String = (args.get(name: "scopeName"))!
```

Comment:
Just an opinion, you can set the scopeName as optional,  so you can choose the default scope by setting scopeName=nil;